server/remote_cache/hit_tracker/hit_tracker.go

```diff
@@ -255,6 +261,36 @@ func (h *HitTracker) TrackUpload(d *repb.Digest) *transferTimer {
 	}
 }
 
+func (h *HitTracker) recordUsage(actionCache bool, d *repb.Digest, actionCounter counterType) error {
+	if h.usage == nil {
+		return nil
+	}
+
+	// If we have an invocation ID, record that the invocation occurred.
+	if h.iid != "" {
```

Comment:
kinda confused why ObserveInvocation is called here as well as inside the invocation handling code, do we need both? (ideally, rm this one)

Reply:
Removed (decided that we won't track invocation usage for cache-only invocations)